### DISCUSSION TITLE
Directus environment for development and production

### DISCUSSION THREAD

**Discussion by kieryk123** [directus/directus#2824] [Archive]
Hi, I am planning to run my new project on Directus CMS which I will self-host.

I am wondering if it's possible to have completely separate environment for development and production. 

How can I do this? Does anyone have a clue?

**Comment by rijkvanzanten** [comment]
You can either use a different project (https://docs.directus.io/guides/projects.html#multitenancy), or a separate installation of Directus

**Reply by michael-scheurer** [reply]
Updated link: https://docs.directus.io/getting-started/glossary/#multitenancy

**Reply by iamsellek** [reply]
@michael-scheurer you're the real mvp thanks for the updated link!!

**Reply by am17torres** [reply]
Now for the new new link :) 
https://docs.directus.io/user-guide/overview/glossary.html#multitenancy
